python/packages/autogen-ext/src/autogen_ext/tools/mcp/_config.py

```diff
@@ -7,7 +8,7 @@
 class StdioServerParams(StdioServerParameters):
     """Parameters for connecting to an MCP server over STDIO."""
 
-    pass
+    stdio_read_timeout: timedelta = timedelta(seconds=10)
```

Comment:
```suggestion
    read_timeout: timedelta = timedelta(seconds=10)
```

nit

Reply:
Also, can we use a simpler `read_timeout_seconds: int` instead. `timedetla` is not JSON serializable and can cause issue when we want to create serializable config for the tool.